packages/now-go/index.ts

```diff
@@ -109,7 +110,7 @@ Learn more: https://github.com/golang/go/wiki/Modules
 Learn more: https://zeit.co/docs/v2/advanced/builders/#go
       `
     );
-    console.log(err.message);
+    debug(err.message);
```

Comment:
```suggestion
    console.log(err.message);
```

Error messages need to always print